pyanaconda/startup_utils.py

```diff
@@ -99,14 +99,14 @@ def check_memory(anaconda, options, display_mode=None):
         display_mode = anaconda.display_mode
 
     reason = reason_strict
-    total_ram = int(isys.total_memory() / 1024)
-    needed_ram = int(isys.MIN_RAM)
-    graphical_ram = int(isys.MIN_GUI_RAM)
+    total_ram = int(hw.total_memory() / 1024)
+    needed_ram = int(hw.MIN_RAM)
+    graphical_ram = int(hw.MIN_GUI_RAM)
 
     # count the squashfs.img in if it is kept in RAM
     if not util.persistent_root_image():
-        needed_ram += isys.SQUASHFS_EXTRA_RAM
-        graphical_ram += isys.SQUASHFS_EXTRA_RAM
+        needed_ram += hw.SQUASHFS_EXTRA_RAM
+        graphical_ram += hw.SQUASHFS_EXTRA_RAM
```

Comment:
I would recommend to drop the `hw.` prefix completely. Renaming these modules is always painful.